arch/arm64/boot/dts/qcom/sdm450-samsung-a6plte-r4.dts

```diff
@@ -592,7 +607,7 @@
 		};
 
 		led-controller {
-			compatible = "siliconmitus,sm5708-fleds";
+			compatible = "siliconmitus,sm5708-fled";
```

Comment:
A possible bug about the charging driver:

The flash LEDs light up only when the USB port is connected with charging or OTG cable.
Similar issue can be seen on `rt5033-charger`.  A possible workaround could be to drop high impedance mode.
However I have no idea how to do this on sm5708.

Link: https://lore.kernel.org/all/6b474e64f752ca6802dfb0bfc5571eb1adad8449.1677620677.git.jahau@rocketmail.com/